java-client/src/main/java/org/opensearch/client/opensearch/core/msearch/MultisearchBody.java

```diff
@@ -202,7 +203,7 @@ public final Query postFilter() {
     /**
      * API name: {@code search_after}
      */
-    public final List<String> searchAfter() {
+    public final List<FieldValue> searchAfter() {
```

Comment:
@Xtansia I am fine having this for 3.x but it is breaking change for 2.x line

Reply:
Actually I discovered that I can just use the `FieldValue` and handle it manually in my code. So I'm not longer blocked. Apologies for the premature comment on my end.

Reply:
@reta The backwards compatible version was already merged into 2.x in #1224, this is just for 3.x/main.

@workmanw There's already a workaround as of version [2.15.0](https://github.com/opensearch-project/opensearch-java/blob/main/CHANGELOG.md#2150---10172024), you can use `searchAfterVals` and `sortVals` to get/set them as `FieldValue`s rather than `String`s.

Reply:
> @reta The backwards compatible version was already merged into 2.x in #1224, this is just for 3.x/main.

:+1: